modules/settings.js

```diff
@@ -12,64 +11,137 @@ function Settings() {
     repeat: true
   };
 
+  // Are these the paths we want?
+  this.backupPaths = {
+    apps: 'backup/apps/',
+    music: 'backup/music/',
+    photos: 'backup/photos/',
+    videos: 'backup/videos/',
+    contacts: 'backup/videos/',
+    settings: 'backup/settings/'
+  };
+
+  // Load persistent settings from local storage, if they exist
+  this.load();
+}
+
+Settings.prototype.load = function() {
   // Load options if present
-  (function() {
-    var retrievedOptions = localStorage.getItem('ffosbrOptions');
+  var retrievedOptions = localStorage.getItem('ffosbrOptions');
 
-    if (retrievedOptions !== null) {
-      self.options = JSON.parse(retrievedOptions);
-    }
-  })();
+  if (retrievedOptions !== null) {
 
-  Settings.prototype.options = function(myDictionary) {
-    if (!myDictionary) {
-      return options;
-    }
+    retrievedOptions = JSON.parse(retrievedOptions);
```

Comment:
If the options object in local storage contains malformed json, what would you like to do next?
I think it's obvious we clear the options object from local storage. But then do you want to throw an error or just continue without storing the loaded options into this.options?
By the way, this same question is posed in a comment pertaining to failed validation. Which we choose, I'll take the same approach for both failures.